components/ws-manager/pkg/manager/imagespec.go

```diff
@@ -70,7 +74,12 @@ func (m *Manager) GetImageSpec(ctx context.Context, req *regapi.GetImageSpecRequ
 		if err != nil {
 			return nil, xerrors.Errorf("cannot unmarshal init config: %w", err)
 		}
-		cl, _, err := m.Content.GetContentLayer(ctx, owner, workspaceID, &initializer)
+		var cl []layer.Layer
+		if pvcFeatureEnabled {
+			cl, _, err = m.Content.GetContentLayerPVC(ctx, owner, workspaceID, &initializer)
+		} else {
+			cl, _, err = m.Content.GetContentLayer(ctx, owner, workspaceID, &initializer)
+		}
```

Comment:
How about raising the abstraction level of the Content layer? In other words, how about installing an interface like ContentLayerProvider, and implementing `Get()` for both the conventional conventional method and the PVC Content method? Is this a bit much?